engine/installation/linux/oracle.md

```diff
@@ -55,7 +64,7 @@ You can install Docker in different ways, depending on your needs:
 
 - Some users cannot use third-party repositories, and must rely on
   the version of Docker in the Oracle repositories. This version of Docker may
-  be out of date. Those users should consult the Oracle documentation and not
+  be out of date. Those users should consult the [Oracle Linux Docker User's Guide](http://docs.oracle.com/cd/E52668_01/E75728/html/index.html) and not
```

Comment:
Please remove this link. It will confuse users of Docker's packages since the Oracle documentation does not mention the existence of Docker's packages.